src/scss/_components.percentage-card.scss

```diff
@@ -0,0 +1,54 @@
+.cmp-percentage-card-grid {
+  @supports (display: grid) {
+    display: grid;
+    grid-gap: 1rem 0;
+
+    @media (min-width: $bp-md) {
+      grid-template-columns: 1fr 1fr 1fr;
+      justify-items: center;
+      grid-gap: 0;
+    }
+  }
+}
+
+.cmp-percentage-card {
+  margin: 2rem auto;
+  text-align: center;
+
+  @supports (display: grid) {
+    display: grid;
+    grid-template-rows: 1fr;
+    justify-items: center;
+    grid-column: span 1;
+    grid-gap: 1rem 0;
+    grid-auto-rows: auto;
+
+    &__icon,
+    &__label,
+    &__percentage {
+      grid-row: span 1;
+    }
+  }
+
+  &__icon {
+    height: 35vw;
```

Comment:
I love that your are making these icons scale with the space. The thing with setting the `height` this way is that the `height` continues to grow even after we’ve hit the `max-width`, which forces the text further away from the icon as the screen size increases. Just as you have a `max-width`, you’ll need a `max-height`. The way this is setup, we should be good setting the `max-height` to `16.25rem` as well.